products/flux2.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Flux
+category: server-app
+tags: cncf
+iconSlug: flux2
```

Comment:
```suggestion
```

There is currently no Flux icon on https://simpleicons.org/?q=flux.

Reply:
Is it mandatory to have an icon there in order for this PR to be merged? I'll issue a PR to simple icons but not sure how quick it'll get merged.

Reply:
[Issue raised](https://github.com/simple-icons/simple-icons/issues/9333)